packages/@lwc/perf-benchmarks/README.md

```diff
@@ -61,3 +61,14 @@ BENCHMARK_TIMEOUT=5
 See the [Tachometer documentation](https://github.com/Polymer/tachometer) for details on what these mean.
 
 If anything gets messed up when comparing to the other branch, add the `--force-clean-npm-install` flag when running `tach`.
+
+Also note that, due to how NX does caching, any environment variables you pass to `build:performance` should also
+be passed to `test:performance`. Otherwise, `test:performance` will use whatever environment variables you give it
```

Comment:
Now that we have NX in place, should the `test:performance` task depends on the `build:performance` task and leverage NX caching to only rebuild when either a benchmark code change or whenever we change an environment variable?

Reply:
`test:performance` does depend on `build:performance`. I don't know why it has this behavior; I haven't explicitly configured it.